### PR TITLE
[Feature] : Work component

### PR DESCRIPTION
**Pull request by SahilVermaFED**
## Description

-   [Ticket Link]

-  Custom CSS styles, addressing font import issues and introducing color names, border radius, width, background 
    color, line height, and padding .
- New assets (logo and images), removes the default React logo .
- GSAP for scroll handling, leading to the creation of 'scrollHandlerUtils.jsx'

## UI screenshot or Video
[work-component.webm](https://github.com/primathonacademy/april-batch-live-project/assets/131689244/502a656f-eae8-4d9e-bf30-ff0f94d4dcfa)

### General:

-   [x] **PR Size :** Confirming PR size < 20 files or pre-approved if > 20 files.
-   [x] **Self Review :** Confirming I have self-reviewed my PR
-   [x] **Peers Review :** - [ ] @Dev 1
-   [x] Confirming I have added all sensitive information-related should be inside .env files

### Client:

-   [x] **Console :** Confirming no console errors in the UI for this update.

-   [x] **Import order :** Confirming correct import order.

```
# External Import
import React, { useState, useEffect } from 'react';
import { useParams } from 'react-router-dom';
import { useDispatch, useSelector, shallowEqual } from 'react-redux';

# Internal Import
import { } from '../../Api/ApiRoutes';
import { } from '../../constants/AppConstant';
```

-   [x] **Re-renders :** Confirming minimal re-renders for new React components.

-   [x] **Network calls :** Confirming efficient network/API calls, minimized as needed.

-   [x] **Util function comments :** Confirming comprehensive comments for utility functions.

-   [x] **Image optimization :** Confirming I have checked that all images are optimized. If not, please use https://tinyjpg.com/ or https://vecta.io/nano.

-   [x] **Dead code :** Confirming I have removed all comments code which is of no use OR written why I have kept the commented code.

### Confirming different viewport testing of the entire flow:

-   [x] Tested on 280px viewport
-   [x] Tested on 320px viewport
-   [x] Tested on 375px viewport
-   [x] Tested on 425px viewport
-   [x] Tested on 768px viewport
-   [x] Tested on 1024px viewport
-   [x] Tested on 1440px viewport
-   [x] Tested on 2560px viewport

### End-to-end testing:

-   [x] **Happy flow :** Successfully tested end-to-end happy flow, all working as expected.
-   [x] **Error flow :** Tested UI flow with error scenarios, gracefully handled errors on UI
-   [x] **Common component changed :** Tested change in common component across views to prevent breaking elsewhere.

### CODE REVIEW

File: src/assets/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export { default as workImg1 } from '../../src/assets/homepage/our-work-1.png';
```
Comment (ram-primathon) [inline src/assets/index.js]:
```suggestion
export { default as workImg1 } from './homepage/our-work-1.png';
```

File: src/assets/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+export { default as workImg2 } from '../../src/assets/homepage/our-work-2.png';
```
Comment (ram-primathon) [inline src/assets/index.js]:
```suggestion
export { default as workImg2 } from './homepage/our-work-2.png';
```

File: src/assets/index.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+export { default as workImg3 } from '../../src/assets/homepage/our-work-3.png';
```
Comment (ram-primathon) [inline src/assets/index.js]:
```suggestion
export { default as workImg3 } from './homepage/our-work-3.png';
```

File: src/assets/index.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+export { default as workImg4 } from '../../src/assets/homepage/our-work-4.png';
```
Comment (ram-primathon) [inline src/assets/index.js]:
```suggestion
export { default as workImg4 } from './homepage/our-work-4.png';
```

**Review by ram-primathon — CHANGES_REQUESTED** [review]
Please fix import path

File: src/assets/index.js [hunk 5]
```diff
@@ -1,4 +1,5 @@
-export { default as instagramIcon } from './icons/icon-instagram.svg';
-export { default as linkedinIcon } from './icons/icon-linkedin.svg';
-export { default as twitterIcon } from './icons/icon-twitter.svg';
-export { default as brandLogo } from './icons/brand-logo.svg';
+export { default as workImg1 } from './homepage/our-work-1.png';
```
Comment (ram-primathon) [inline src/assets/index.js:1]: Why have you deleted this?

File: src/pages/homepage/WorkComponent.jsx [hunk 6]
```diff
@@ -0,0 +1,137 @@
+import { Fragment, useRef } from 'react';
+import { useGSAP } from '@gsap/react';
+import { NavLink } from 'react-router-dom';
+import portfolioArrow from '../../../public/icons/arrow-blue-circle.svg';
+import scrollHandler from '../../utils/scrollHandlerUtils';
+import { workImg1, workImg2, workImg3, workImg4 } from '../../assets/index';
```
Comment (ram-primathon) [inline src/pages/homepage/WorkComponent.jsx]:
```suggestion
import { workImg1, workImg2, workImg3, workImg4 } from '../../assets';
```

File: src/pages/homepage/WorkComponent.jsx [hunk 7]
```diff
@@ -0,0 +1,137 @@
+import { Fragment, useRef } from 'react';
+import { useGSAP } from '@gsap/react';
+import { NavLink } from 'react-router-dom';
+import portfolioArrow from '../../../public/icons/arrow-blue-circle.svg';
```
Comment (ram-primathon) [inline src/pages/homepage/WorkComponent.jsx]: This is wrong

File: src/pages/homepage/index.jsx [hunk 8]
```diff
@@ -1,5 +1,11 @@
+import WorkComponent from './WorkComponent';
+
 const Homepage = () => {
-    return <div className="px-6 md:px-16 lg:px-20 xl:px-28 2xl:px-36">Welcome to Homepage</div>;
+    return (
+        <div>
```
Comment (ram-primathon) [inline src/pages/homepage/index.jsx]: Please remove this div

File: src/assets/index.js [hunk 9]
```diff
@@ -1,10 +1,14 @@
...
+export { default as workImg4 } from './homepage/our-work-4.png';
+export { default as ourServiceImage1 } from './images/our-service-1.png';
+export { default as ourServiceImage2 } from './images/our-service-2.png';
+export { default as ourServiceImage3 } from './images/our-service-3.png';
+export { default as rightArrowIcon } from './images/right-arrow-icon.png';
 export { default as instagramIcon } from './icons/icon-instagram.svg';
 export { default as linkedinIcon } from './icons/icon-linkedin.svg';
 export { default as twitterIcon } from './icons/icon-twitter.svg';
 export { default as brandLogo } from './icons/brand-logo.svg';
+export { default as forwardArrow } from './icons/forward-arrow.svg';
```
Comment (ram-primathon) [inline src/assets/index.js]: is forwardArrow and rightArrow not same?

**Review by ram-primathon — CHANGES_REQUESTED** [review]

File: src/assets/index.js (hunk 9 above)
Comment (TheSahilVerma) [reply src/assets/index.js]: forwardArrow is arrow with blue background and rightArrow is only an arrow

File: src/assets/index.js (hunk 5 above)
Comment (TheSahilVerma) [reply src/assets/index.js:1]: these are added below in line no 5,6,7,8

**Review by ram-primathon — APPROVED** [review]

**Pull request merged**